### PR TITLE
gui: /rest/system/browse with no arguments returns drives on Windows …

### PR DESCRIPTION
**Pull request by AudriusButkevicius**
See title

### CODE REVIEW

File: lib/osutil/drives_windows.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"unsafe"
+)
+
+func GetDriveLetters() []string {
+	drives := []string{}
+
+	kernel32, err := syscall.LoadDLL("kernel32.dll")
+	if err != nil {
+		return drives
+	}
```
Comment (calmh) [inline lib/osutil/drives_windows.go]: Why not just return nil?

Comment (AudriusButkevicius) [inline lib/osutil/drives_windows.go]: doesn't really matter does it?

Comment (calmh) [inline lib/osutil/drives_windows.go]: It matters only in that when looking at this I need to keep track of if we've actually done anything with `drives` to see what we return in the error cases. That's trivial in the first case but when looking at line 34 I actually have to parse lines 18-33 as well to understand what it means.

As an aside, shouldn't we just return `([]string, error)` and let the API return a 500 when this fails on Windows? The empty success response isn't meaningful by itself, is it?

**Comment by AudriusButkevicius** [comment]
bump.

**Comment by AudriusButkevicius** [comment]
Bump.

**Comment by calmh** [comment]
@st-review merge it!

**Comment by st-review** [comment]
:ok_hand: Merged as 1612baca9203c3029b6818e40ceb9f195a7da205. Thanks, @AudriusButkevicius!

**Pull request closed without merging**